dids/src/test/resources/did_jwk_x25519_document.json

```diff
@@ -1,13 +1,10 @@
 {
-  "@context": [
-    "https://www.w3.org/ns/did/v1",
-    "https://w3id.org/security/suites/jws-2020/v1"
-  ],
+  "@context": "https://www.w3.org/ns/did/v1",
   "id": "did:jwk:eyJrdHkiOiJPS1AiLCJjcnYiOiJYMjU1MTkiLCJ1c2UiOiJlbmMiLCJ4IjoiM3A3YmZYdDl3YlRUVzJIQzdPUTFOei1EUThoYmVHZE5yZngtRkctSUswOCJ9",
   "verificationMethod": [
     {
       "id": "did:jwk:eyJrdHkiOiJPS1AiLCJjcnYiOiJYMjU1MTkiLCJ1c2UiOiJlbmMiLCJ4IjoiM3A3YmZYdDl3YlRUVzJIQzdPUTFOei1EUThoYmVHZE5yZngtRkctSUswOCJ9#0",
-      "type": "JsonWebKey2020",
+      "type": "JsonWebKey",
```

Comment:
I believe this should be reverted, but practically it doesn't matter as long as we're internally consistent

the original did jwk spec was created before the `JsonWebKey` type existed

Reply:
yea. i did this coz i thought we were internally moving towards aligning on JsonWebKey, so i changed the type for verificationmethod of DidJwk during resolve() to be `JsonWebKey` - https://github.com/TBD54566975/web5-kt/pull/248/files#diff-5537230f4c2b63c726999e7c345be0afee8bad05f7d49154d9897013f550ae7aR152

should i keep it as JsonWebKey2020? then i can revert this change.